ambry-clustermap/src/main/java/com/github/ambry/clustermap/DataNodeConfig.java

```diff
@@ -127,6 +128,14 @@ Set<String> getSealedReplicas() {
     return sealedReplicas;
   }
 
+  /**
+   * @return the set of partially sealed replicas on this server. This set is mutable. Replicas marked as partially
+   * sealed will not be able to take writes for metadata chunks whose id is reserved at the start of the upload.
```

Comment:
Nit: As you and Jing were discussing, may be remove mentioning of metadata chunks?

Reply:
fixed.